ui/articles-aem-renderer/src/main/java/org/cru/godtools/articles/aem/service/OkHttpClientProvider.java

```diff
@@ -0,0 +1,122 @@
+package org.cru.godtools.articles.aem.service;
+
+import android.os.Build;
+
+import java.io.IOException;
+import java.net.InetAddress;
+import java.net.Socket;
+import java.security.KeyManagementException;
+import java.security.KeyStore;
+import java.security.NoSuchAlgorithmException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+
+import javax.net.ssl.SSLContext;
+import javax.net.ssl.SSLSocket;
+import javax.net.ssl.SSLSocketFactory;
+import javax.net.ssl.TrustManager;
+import javax.net.ssl.TrustManagerFactory;
+import javax.net.ssl.X509TrustManager;
+
+import okhttp3.ConnectionSpec;
+import okhttp3.OkHttpClient;
+import timber.log.Timber;
+
+public class OkHttpClientProvider {
+
+    public static OkHttpClient.Builder createBuilder() {
```

Comment:
the builder I was referencing was a brand new builder for the SSLSocketFactory only. How `OkHttpClient` is created can stay completely within the `AemApi` method that originally had it.